src/manifolds/MetricManifold.jl

```diff
@@ -51,6 +51,129 @@ inner product ``g(X, X) > 0`` whenever ``X`` is not the zero vector.
 """
 abstract type RiemannianMetric <: AbstractMetric end
 
+@doc raw"""
+    change_gradient(M::AbstractManifold, G2::AbstractMetric, p, X)
+
+Convert the gradient `X` at `p` on the[`AbstractManifold`](@ref) `M` from one metric to another.
+
+Assume that for a real-valued function ``f: \mathcal M \to ℝ`` the input `X` is the gradient or in
+other words the [Riesz representer](https://en.wikipedia.org/wiki/Riesz_representation_theorem#Riesz_representation_theorem) of the differential ``Df(p)``` with respect to the metric ``g_2`` i.e.
+
+```math
+    g_2(X,Y) = Df(p)[Y] \quad \text{for all } Y ∈ T_p\mathcal M.
+```
+
+(It could actually also be the Riesz representer of _any_ linear function, not just ``Df(p)``).
+
+In order to convert this into the gradient with respect to the (implicitly given) metric ``g_1`` of `M`,
+we have to find the conversion function ``c: T_p\mathcal M \to \T_p\mathcal M`` such that
+
+```math
+    g_2(X,Y) = g_1(c(X),Y)
+```
+
+If both metrics are given in their [`local_metric`](@ref) (symmetric positive defintie) matrix
+representations ``G_1`` and ``G_2`` and ``x,y`` are the local coordinates with respect to
+the same basis of the tangent space, the equation reads
+
+```math
+   x^{\mathrm{H}}G_2y = c(x)^{\mathrm{H}}G_1 y \quad \text{for all } y \in ℝ^d,
+```
+where `\cdot^{\mathrm{H}}`` denotes the conjugate transpose.
+
+and we obtain ``c(X) = (G_1\backslash G_2)^{\mathrm{H}X``
+
+# Examples
+
+    change_gradient(Sphere(2), EuclideanMetric(), p, X)
+
+Since the metric in ``T_p\mathbb S^2`` is the Euclidean metric from the embedding restricted to ``T_p\mathbb S^2``, this just returns `X`
+
+    change_gradient(SymmetricPOsitiveDefinite(3), EuclideanMetric(), p, X)
+
+Here, the default metric in `\mathcal P(3)` is the [`LinearAffineMetric`](@ref) and the transformation can be computed as ``pXp``
+"""
+change_gradient(::AbstractManifold, ::AbstractMetric, ::Any, ::Any)
+
+function change_tangent(M::AbstractManifold, G::AbstractMetric, p, X)
+    if is_default_metric(M, G)
+        return X
+    end
+    # TODO: For local metric, inverse_local metric, det_local_metric: Introduce a default basis?
+    B = DefaultOrthogonalBasis()
+    G1 = local_metric(M, p, B)
+    G2 = local_metric(G(M), p, B)
+    x = get_coordinates(M, p, X, B)
+    z = (G2 \ G1)'x
+    return get_vector(M, p, z, B)
+end
+
+function change_metric(
+    ::MetricManifold{𝔽,M,G},
+    ::G,
+    p,
+    X,
+) where {𝔽,M<:AbstractManifold{𝔽},G<:AbstractMetric}
+    return X
+end
+
+@doc raw"""
+    change_metric(M::AbstractcManifold, G2::AbstractMetric, p, X)
+
+On the [`AbstractManifold`](@ref) `M` with implicitly given metric ``g_1``
+and a second [`AbstractMetric`](@ref) ``g_2`` this method performs a change of metric in the
+sense that it returns the tangent vector ``Z=BX`` such that the linear map ``B`` fulfills
+
+````math
+g_2(Y_1,Y_2) = g_1(BY_1,BY_2) \quad \text{for all } Y_1, Y_2 ∈ T_p\mathcal M.
+````
+
+holds.
```

Comment:
Having both "holds" and "fulfills" seems a bit excessive.

Reply:
oh yes, we can leave out the “holds”, sure.